app/assets/stylesheets/active_admin_sidebar_pure_icons.scss

```diff
@@ -0,0 +1,62 @@
+/* .icono-caret-left */
+.icono-caret-right,
+.icono-caret-left {
+  width: 19px;
+  height: 19px;
+}
+.icono-caret-right:before,
+.icono-caret-right:after,
+.icono-caret-left:before,
+.icono-caret-left:after {
+  width: 8px;
+  height: 2px;
+  position: absolute;
+  bottom: 1px;
+  margin: auto 0;
+  right: 6px;
+  box-shadow: inset 0 0 0 32px;
+  transform-origin: right;
+}
+.icono-caret-right:before,
+.icono-caret-left:before {
+  top: 2px;
+  -webkit-transform: rotate(45deg);
```

Comment:
Properties should be ordered top, -moz-transform, -ms-transform, -o-transform, -webkit-transform, transform<br>Avoid vendor prefixes.